sdk/communication/Azure.Communication.Administration/src/PhoneNumberAdministrationClient.cs

```diff
@@ -68,7 +68,7 @@ public virtual AsyncPageable<AcquiredPhoneNumber> GetAllPhoneNumbersAsync(string
                 return PageResponseEnumerator.CreateAsyncEnumerable(async s =>
                 {
                     Response<AcquiredPhoneNumbers> response = await RestClient.GetAllPhoneNumbersAsync(locale, skip: null, take: null, cancellationToken).ConfigureAwait(false);
-                    return Page.FromValues(response.Value.PhoneNumbers, continuationToken: null!, response.GetRawResponse());
+                    return Page.FromValues(response.Value.PhoneNumbers, response.Value.NextLink, response.GetRawResponse());
```

Comment:
Let's add this to the changelog under a Key bug fixes heading for the next unreleased version.